javascripting/array-filtering.js

```diff
@@ -0,0 +1,7 @@
+var numbers = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10];
+
+var filtered = numbers.filter(function(number) {
+  return number % 2 === 0;
```

Comment:
Fair enough. I was only talking about its (==) usage in this rather contrived example because the underlying JS engine doesn't perform type-coercion in this particular case.

However, for always all production use cases I like to use `===`. Explicit is always better than implicit.